net-im/skypeforlinux/skypeforlinux-8.34.0.78-r3.ebuild

```diff
@@ -0,0 +1,119 @@
+QA_PREBUILT="*"
+RESTRICT="mirror bindist strip" #299368
+
+RDEPEND="
+	|| (
+		sys-auth/elogind
+		sys-apps/systemd
+	)
+	dev-libs/atk[${MULTILIB_USEDEP}]
+	dev-libs/expat[${MULTILIB_USEDEP}]
+	dev-libs/glib:2[${MULTILIB_USEDEP}]
+	dev-libs/nspr[${MULTILIB_USEDEP}]
+	dev-libs/nss[${MULTILIB_USEDEP}]
+	gnome-base/gconf:2[policykit,${MULTILIB_USEDEP}]
+	media-libs/alsa-lib[${MULTILIB_USEDEP}]
+	media-libs/fontconfig:1.0[${MULTILIB_USEDEP}]
+	media-libs/freetype:2[${MULTILIB_USEDEP}]
+	media-libs/libv4l[${MULTILIB_USEDEP}]
+	net-print/cups[${MULTILIB_USEDEP}]
+	sys-apps/dbus[${MULTILIB_USEDEP}]
+	sys-devel/gcc[cxx]
+	virtual/ttf-fonts
+	x11-libs/cairo[${MULTILIB_USEDEP}]
+	x11-libs/gdk-pixbuf:2[${MULTILIB_USEDEP}]
+	x11-libs/gtk+:3[${MULTILIB_USEDEP}]
+	x11-libs/libX11[${MULTILIB_USEDEP}]
+	x11-libs/libXScrnSaver[${MULTILIB_USEDEP}]
+	x11-libs/libXcomposite[${MULTILIB_USEDEP}]
+	x11-libs/libXcursor[${MULTILIB_USEDEP}]
+	x11-libs/libXdamage[${MULTILIB_USEDEP}]
+	x11-libs/libXext[${MULTILIB_USEDEP}]
+	x11-libs/libXfixes[${MULTILIB_USEDEP}]
+	x11-libs/libXi[${MULTILIB_USEDEP}]
+	x11-libs/libXrandr[${MULTILIB_USEDEP}]
+	x11-libs/libXrender[${MULTILIB_USEDEP}]
+	x11-libs/libXtst[${MULTILIB_USEDEP}]
+	x11-libs/libxcb[${MULTILIB_USEDEP}]
+	x11-libs/libxkbfile[${MULTILIB_USEDEP}]
+	x11-libs/pango[${MULTILIB_USEDEP}]"
+
+src_unpack() {
+	rpm_src_unpack ${A}
+}
+
+src_prepare() {
+	default
+	sed -e "s!^SKYPE_PATH=.*!SKYPE_PATH=${EPREFIX}/opt/skypeforlinux/skypeforlinux!" \
+		-i usr/bin/skypeforlinux || die
+	sed -e "s!^Exec=/usr/!Exec=${EPREFIX}/opt/!" \
+		-e "s!^Categories=.*!Categories=Network;InstantMessaging;Telephony;!" \
+		-e "/^OnlyShowIn=/d" \
+		-i usr/share/applications/skypeforlinux.desktop || die
+}
+
+src_install() {
+	dodir /opt
+	cp -a usr/share/skypeforlinux "${D}"/opt || die
+
+	into /opt
+	dobin usr/bin/skypeforlinux
+
+	dodoc usr/share/skypeforlinux/*.html
+	dodoc -r usr/share/doc/skypeforlinux/.
+	# symlink required for the "Help->3rd Party Notes" menu entry  (otherwise frozen skype -> xdg-open)
+	dosym ${PF} usr/share/doc/skypeforlinux
+
+	doicon usr/share/pixmaps/skypeforlinux.png
+
+	# compat symlink for the autostart desktop file
+	dosym ../../opt/bin/skypeforlinux usr/bin/skypeforlinux
+
+	local res
+	for res in 16 32 256 512; do
+		newicon -s ${res} usr/share/icons/hicolor/${res}x${res}/apps/skypeforlinux.png skypeforlinux.png
+	done
+
+	domenu usr/share/applications/skypeforlinux.desktop
+
+	if use pax_kernel; then
+		pax-mark -m "${ED%/}"/opt/skypeforlinux/skypeforlinux
+		pax-mark -m "${ED%/}"/opt/skypeforlinux/resources/app.asar.unpacked/node_modules/slimcore/bin/slimcore.node
+		ewarn "You have set USE=pax_kernel meaning that you intend to run"
+		ewarn "${PN} under a PaX enabled kernel. To do so, we must modify"
+		ewarn "the ${PN} binary itself and this *may* lead to breakage! If"
+		ewarn "you suspect that ${PN} is being broken by this modification,"
+		ewarn "please open a bug."
+	fi
+}
+
+pkg_postinst() {
```

Comment:
can you add the `ewarn` message line here for enabling a `sys-auth/elogind` service on the `openrc`?

Reply:
Is this actually needed? The elogind ebuild already prints messages about this. It says that enabling the service is not actually needed and that it will be started automatically:

```
pkg_postinst() {
        if [[ "$(rc-config list boot | grep elogind)" != "" ]]; then
                elog "elogind is currently started from boot runlevel."
        elif [[ "$(rc-config list default | grep elogind)" != "" ]]; then
                ewarn "elogind is currently started from default runlevel."
                ewarn "Please remove elogind from the default runlevel and"
                ewarn "add it to the boot runlevel by:"
                ewarn "# rc-update del elogind default"
                ewarn "# rc-update add elogind boot"
        else
                elog "elogind is currently not started from any runlevel."
                elog "You may add it to the boot runlevel by:"
                elog "# rc-update add elogind boot"
                elog
                elog "Alternatively, you can leave elogind out of any"
                elog "runlevel. It will then be started automatically"
                if use pam; then
                        elog "when the first service calls it via dbus, or"
                        elog "the first user logs into the system."
                else
                        elog "when the first service calls it via dbus."
                fi
        fi
}
```